cmd/image-builder/config.go

```diff
@@ -102,3 +105,141 @@ func (r *Registry) UnmarshalYAML(value *yaml.Node) error {
 	*r = regs
 	return nil
 }
+
+// GitStateConfig holds information about repository and specific commit
+// from which image should be build.
+// It also contains information wheter job is presubmit or postsubmit
+type GitStateConfig struct {
+	// Name of the source repository
+	RepositoryName string
+	// Name of the source repository's owner
+	RepositoryOwner string
+	// Type of the job, allowed values "presubmit" or "postsubmit"
+	JobType string
+	// Number of the pull request for presubmit job
+	PullRequestNumber string
```

Comment:
Can we use an int type instead string?

Reply:
Sure, I initially took types from env vars, but conversion will be easy